packages/mermaid/src/diagrams/sankey/sankeyRenderer.ts

```diff
@@ -0,0 +1,194 @@
+// @ts-nocheck TODO: fix file
+import { Diagram } from '../../Diagram.js';
+import * as configApi from '../../config.js';
+
+import {
+  select as d3select,
+  // @ts-ignore TODO: make proper import
+  scaleOrdinal as d3scaleOrdinal,
+  // @ts-ignore TODO: make proper import
+  schemeTableau10 as d3schemeTableau10,
```

Comment:
Installing `@types/d3-scale-chromatic` and `@types/d3-scale` should resolve these issues

Reply:
Done